opensearch/src/main/java/org/opensearch/sql/opensearch/request/AggregateAnalyzer.java

```diff
@@ -383,6 +419,12 @@ yield switch (functionName) {
     };
   }
 
+  private static boolean isStringType(ExprType fieldType) {
+    return fieldType instanceof OpenSearchTextType
+        || fieldType == ExprCoreType.STRING
+        || fieldType == ExprCoreType.IP;
+  }
+
```

Comment:
I'm thinking should we do the opposite. Because OpenSearch doc says min/max aggregation can only support numerical fields, but it doesn't mention what field type is supported / not supported by top hits aggregation.

Could you double check what field type min/max aggregation can support, e.g., date, IP, object etc? If that's the case, probably we should check numeric type instead and do top hits aggregation for any non-numerical types?